scripts/sanitycheck

```diff
@@ -3615,19 +3615,21 @@ def main():
         filtered = []
         for d in serial_devices:
             if d.manufacturer in ['ARM', 'SEGGER', 'MBED', 'STMicroelectronics',
-                                  'Atmel Corp.', 'Texas Instruments']:
+                                  'Atmel Corp.', 'Texas Instruments',
+                                  'Silicon Labs', 'NXP Semiconductors']:
                 # TI XDS110 can have multiple serial devices for a single board
                 # assume endpoint 0 is the serial, skip all others
                 if d.manufacturer == 'Texas Instruments' and not d.location.endswith('0'):
                     continue
                 s_dev = {}
                 s_dev['platform'] = "unknown"
                 s_dev['id'] = d.serial_number
+                s_dev['runner-id'] = d.serial_number
```

Comment:
is runner-id a good name here? might be confused with the runner key we have below.

Reply:
I think it's a good name: it specifies the `id` that's to be used in association with the `runner`.  Do you have an alternative?

Reply:
Maybe device_id, alt_id, external_id, probe_id...
I would also not populate it with the serial_number from scanning, leave it empty or make it optional, so a user can add it, if it specified, the it will be used instead of 'id'.

Reply:
I went with "probe_id" as that seemed to make sense to be paired with the runner and would be less likely to be confused with an id associated with the board itself.

Reply:
> I would also not populate it with the serial_number from scanning, leave it empty or make it optional, so a user can add it, if it specified, the it will be used instead of 'id'.

what about the this?

Reply:
I removed that in the [same update](https://github.com/zephyrproject-rtos/zephyr/compare/b1ab67b1280ebee638f0a3463760ff4918782587..919e76be7cca2781a44d9cbe22239a8c1bd63ec2) where I changed the name.

Reply:
oops, sorry.